automatic/lightscreen/tools/install.ahk

```diff
@@ -4,7 +4,7 @@ SendMode Input
 SetWorkingDir %A_ScriptDir%
 
 WinActivate, Options - Lightscreen
-Send, {Enter}
+ControlSend, WinTitle, {Enter}
```

Comment:
`WinTitle` should be removed, i.e. should be empty to force the command to use [LastWindowFound](https://autohotkey.com/docs/misc/WinTitle.htm#LastFoundWindow)